analytics_automated/models.py

```diff
@@ -244,6 +244,19 @@ class Meta:
         app_label = 'analytics_automated'
 
 
+class Requirement(models.Model):
```

Comment:
isn't this creating a new table for task only? @waleedev01

Reply:
Yes in this case I create a table called Requirement to aviod long reuqirement json payload. We should now use attribute instead. I will cancel this request and resubmit one